sdk/spring/spring-messaging-azure/src/main/java/com/azure/spring/messaging/container/AbstractListenerContainer.java

```diff
@@ -15,8 +15,6 @@
     private String destination;
     private String group;
     private AzureMessageHandler messageHandler;
-    private boolean autoStartup = true;
```

Comment:
why do we remove these two members?

Reply:
This is used to remove the EnforceFinalFieldsCheck https://github.com/Azure/azure-sdk-for-java/blob/2b38ab0fd1a5520125fba667c329cc3c273ff1d7/eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml#L496, 
Field "phase" is not a ssigned in constructor or methods.You should consider making the field final, or suppressing the warning. [EnforceFinalFields]
I check it's not necessary to override the method because there's no `setXXX`method to update the values, so I decide it can be removed.